doc/sphinx/source/howto/tune-for-deployment.rst

```diff
@@ -0,0 +1,171 @@
+.. tune-for-deployment.rst
+
+
+############################################
+Fine-tune code deployment to nGraph devices  
+############################################
+
+When deploying code to nGraph-enabled devices, there are some additional 
+configuration options that can be incorporated. Fine-tuning an nGraph-enabled
+device is as much of an art as it is a science; there are virtually limitless
+ways to command your training or inference computations to run the way you want.  
+
+Most of the time, and how this has worked :doc:`for many <../framework-integration-guides>` 
+of the "direct optimizations" we've shared with the developer community, 
+`engineering teams spend several weeks or months`_ working on extracting best 
+performance from a specific :abbr:`DL (Deep Learning)` model embedded in a 
+specific framework that is training a specific dataset. Some of the ways we 
+attempt to extract performance include: 
+
+* Testing and recording the results of various system-level configuration options
+  or flags enabled or disabled,
+* Compiling with a mix of custom environment variables, 
+* Finding semi-related comparisons for benchmarking [#1]_, and 
+* Writing new assembly code to help a machine-learning algorithm learn faster or 
+  more accurately that it did on previous runs. 
+
+In nearly every case, the "best" mix of configuration options boils down to 
+something unique to that framework-model topology. The larger and more complex a 
+framework is, the harder it becomes to extract the best performance; default-enabled
+configuration options from the framework side can slow down compilation time 
+without the developer being any the wiser. Sometimes only `a few small`_ 
+adjustments can increase performance. Likewise, a minimally-designed framework
+could offer significant performance-improvement opportunities by lowering overhead.
+
+For this reason, we're providing some of the more commonly-used options for 
+fine-tuning various kinds code deployments to the nGraph-enabled devices we 
+currently support. Watch this section for new updates. 
+
+
+Training Deep Neural Networks
+==============================
+
+This section details some of the *configuration options* and *environment variables* 
+that can be used to tune for optimal performance when your system already has a
+version of nGraph installed.  
+
+Before tweaking various environment variables, be aware that how the computation 
+gets deployed depends upon the data format that the model is using. NHWC or NCHW
+are the two more common layouts, and the ultimate runtime can vary greatly -- 
+even when all other factors are the same -- when this detail is overlooked.
+
+* **N** -- Number of images per batch
+* **H** -- Height of the image
+* **W** -- Width of the image
+* **C** -- Channel of the image (expressed as a number like 3 for RGB and 1 
+  for grayscale)
+
```

Comment:
We need to say if one or the other is preferred. Otherwise, why are we mentioning this? For MKLDNN/CPU backends, we prefer NCHW. So does the GPU, i believe

Reply:
Fixed